std_msgs/String task
---
zeabus_vision/vision_shoot_craps data
